.shippable.yml

```diff
@@ -65,11 +66,18 @@ build:
             ./scripts/ci/check-compliance.py --commits ${COMMIT_RANGE} || true;
           fi;
       - >
+          python2 ./scripts/ci/get_modified_tests.py --commits origin/${PULL_REQUEST_BASE_BRANCH}..HEAD;
```

Comment:
realised I am missing sh for python3, need to update the docker image at some point, will add a FIXME in there.